### PR TITLE
Updated examples and docs to latest versions

### PR DESCRIPTION
**Pull request by jwngr**
https://twitter.com/_jwngr/status/765755705081098240

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,34 +39,45 @@ Firepad uses [Firebase](https://firebase.google.com) as a backend, so it require
...
-
-<!-- Firepad -->
-<link rel="stylesheet" href="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.css" />
-<script src="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.min.js"></script>
+<head>
+  <!-- Firebase -->
+  <script src="https://www.gstatic.com/firebasejs/3.3.0/firebase.js"></script>
+
+  <!-- CodeMirror -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/codemirror.js"></script>
```
Comment (jwngr) [inline README.md]: I upgraded this to the latest version and it all seems to work fine... All the demos worked for what I tested and in theory if they follow semver, we should be all good updating this.

File: examples/hammer.html [hunk 2]
```diff
@@ -5,12 +5,15 @@
   <meta charset="utf-8" />
```
Comment (jwngr) [inline examples/hammer.html]: I have no clue what exactly this example does, but I love it! 🔨

**Comment by jwngr** [comment]
The tests pass for me locally, although they failed the very first time I ran them after cloning. They passed upon re-running them. So I assume something weird is going on there. Gonna move on to other PRs, but let me know if you have any ideas for what's up with the three failing integration tests.

**Comment by mikelehen** [comment]
Hrm.  I see the same thing.  3 tests failed the first time, but then pass consistently afterwards. :-/  I suspect it's related to phantomjs bugginess or slowness.  Upgrading or increasing timeouts might help. Not going to worry about it for now.

Thanks for updating everything!  Looks good.

**Pull request merged**